setup.cfg

```diff
@@ -24,7 +24,7 @@ combine_as_imports = True
 sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
 known_third_party = six, docutils, pkg_resources
 known_zope =
-known_first_party = Products.CMFCore
+known_first_party =Products.CMFCore
```

Comment:
This is a little annoying. I made a small tweak in meta/config to fix that. But that fix leads to a blank space at line end if no values are set.  

Reply:
I improved the fix in https://github.com/zopefoundation/meta/pull/190 to prevent trailing whitespace in case the value is empty.